DocumentsFromSnapshotMigration/src/main/java/org/opensearch/migrations/RfsMigrateDocuments.java

```diff
@@ -502,11 +562,22 @@ private static void executeCleanShutdownProcess(
             return;
         }
         var workItemAndDuration = workItemRef.get();
-        log.atInfo().setMessage("Marking progress: " + workItemAndDuration.getWorkItem().toString() + ", at doc " + progressCursor.get().getProgressCheckpointNum()).log();
```

Comment:
did you get a warning for this line?  I had thought that with ErrorProne these were supposed to be build errors.  I'm wondering how it persisted